tests/unit-tests/data_utils.py

```diff
@@ -248,6 +248,8 @@ def generate_dummy_hetero_graph_reconstruct(size='tiny', gen_mask=True):
                              th.randint(data_size, (data_size,))),
         ("n4", "r3", "n2"): (th.randint(data_size, (data_size,)),
                              th.randint(data_size, (data_size,))),
+        ("n1", "r5", "n2"): (th.randint(data_size, (data_size,)),
```

Comment:
This can trigger the bug? Why?

Reply:
we want to generate features on n2. the bug happens when the target nodes have two relation types. one relation connects neighbors with features and the other doesn't.